.github/workflows/linters.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Install golangci-lint 
       run: |
         go version
-        curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.23.8
+        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.43.0
```

Comment:
Can you please add a reference to any information(if available) which suggests guidance on using this website from now onwards?

Reply:
https://golangci-lint.run/usage/install/#other-ci